google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -245,23 +379,53 @@ def _get_paragraphs(
             from the document.
     Returns:
         List[Paragraph]:
-             A list of Paragraphs.
+             A list of wrapped Paragraphs.
     """
     result = []
 
     for paragraph in paragraphs:
         result.append(
             Paragraph(
-                documentai_paragraph=paragraph,
+                documentai_object=paragraph,
                 text=_text_from_layout(layout=paragraph.layout, text=text),
+                _page=_page,
+            )
+        )
+
+    return result
+
+
+def _get_tokens(tokens: List[documentai.Document.Page.Token], text: str) -> List[Token]:
```

Comment:
Should this get a `page` instead, like the other helper functions?

Reply:
No since this doesn't have children so there is no need to pass in a page.

Reply:
For consistency perhaps let's use `page` here instead of text.  I understand this is passing in more information than needed, but this removes a special case among a family of helper functions.